test/test-pipe-set-fchmod.c

```diff
@@ -0,0 +1,30 @@
+#include "uv.h"
+#include "task.h"
+
+TEST_IMPL(pipe_set_fchmod) {
+  uv_pipe_t pipe_handle;
+  uv_loop_t* loop;
+  int r;
+
+  loop = uv_default_loop();
+
+  r = uv_pipe_init(loop, &pipe_handle, 0);
+  ASSERT(r == 0);
+
+  r = uv_pipe_bind(&pipe_handle, TEST_PIPENAME);
```

Comment:
Other pipe tests don't test if the pipe is available, they will all fail together then.